ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
```

Comment:
I'm probably too pragmatic for some, but this is how I would have tested this file:

```js
import React from 'react';
import { render, mount } from 'enzyme';
import sinon from 'sinon';
import { commonHtmlProps } from '../../test/common_html_props';

import {
  KuiToolBarSearchBox,
} from './tool_bar_search_box';

const noop = () => {};

test('renders KuiToolBarSearchBox', () => {
  const component = <KuiToolBarSearchBox
    onFilter={noop}
    filter="My Query"
    { ...commonHtmlProps } />;
  expect(render(component)).toMatchSnapshot();
});

test('ensure onFilter is called when input changes', () => {
  const onFilter = sinon.spy();
  const searchBox = mount(<KuiToolBarSearchBox onFilter={onFilter} />);

  const event = { target: { value: 'a' } };
  searchBox.find('input').simulate('change', event);

  sinon.assert.calledOnce(onFilter);
  sinon.assert.calledWith(onFilter, 'a');
});
```

I'm not a fan of deep `describe` hierarchies.

(There's also no need for a outer describe in jest, it's already grouped by file when viewed)